scripts/install_composable.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+echo "Composable Multiplatform installer"
+myos=$(uname -a)
+echo $myos
+
+# Ubuntu:
+if echo "$myos" | fgrep -q Ubuntu 2>/dev/null ; then      
+	apt update && apt install -y git clang curl libssl-dev llvm libudev-dev
+# Debian:
+elif echo "$myos" | fgrep -q Debian 2>/dev/null; then
+	apt update && apt install -y git clang curl libssl-dev llvm libudev-dev
+# Fedora
+elif echo "$myos" | fgrep -q Fedora 2>/dev/null; then
+	dnf update && dnf install clang curl git openssl-devel
+# Arch
+elif echo "$myos" | fgrep -q Arch 2>/dev/null; then
+	pacman -Syu --needed --noconfirm curl git clang
+
+# OpenBSD
+#$ pkg_info -Q rust 
```

Comment:
what is these commented lines about? are these untested or optional? may be just remove it?

Reply:
removed them, those were the supported rust packages in openbsd